src/events/models/Event.ts

```diff
@@ -111,46 +109,10 @@ export enum EventView {
   CALENDAR,
 }
 
-export interface IFrontpageEvent {
-  eventUrl: string;
-  startDate: string;
-  author: IUser;
-  title: string;
-  event_start: Date;
-  event_end: Date;
-  ingress: string; // TextField type?
-  description: string; // TextField type?
-  images: IImage[]; // StaticContent URL?
-  event_type: EventType;
-}
-
-export interface IEvent {
-  author: IUser;
-  title: string;
-  event_start: Date;
-  event_end: Date;
-  location: string; // Location type?
-  ingress_short: string; // TextField type?
-  ingress: string; // TextField type?
-  description: string; // TextField type?
-  image: string; // StaticContent URL?
-  event_type: EventType;
-  organizer: IGroup;
-
-  // feedback: Feedback
-}
-
-export interface ICompanyEvent extends IEvent {
+export interface ICompanyEvent {
   company: ICompany;
```

Comment:
This is probably way outside the scope of this PR, but why is the CompanyEvent even it's own type, and not just a link to the company directly? Also wouldn't it make more sense for it to be called organisation, so that you could add collaboration with other organisations, without it being a company event? (Again way outside the scope of this PR)

Reply:
My guess is that it comes from how `ManyToMany` relations work.
There is a table of events can be related to many companies, and the table of companies can be related to many events. To make this work, we create a table of 'CompanyEvents', which links them together.

For simple usage a ManyToManyField would work, but we have custom logic in the `save` method of the model: [example](https://github.com/dotkom/onlineweb4/blob/develop/apps/events/models.py#L854-L860)